.vortex/cli/src/Utils/Tui.php

```
@@ -46,35 +31,61 @@ public static function output(): OutputInterface {
 
   public static function success(string $message): void {
-    info($message);
+    static::marker('✓', $message, static::green(...));
   }
 
   public static function error(string $message): void {
-    error('✕ ' . $message);
+    static::marker('✕', $message, static::yellow(...));
   }
 
   public static function confirm(string $label, bool $default = TRUE, ?string $hint = NULL): bool {
```

PHPMD (2.15.0)

[error] The method confirm has a boolean flag argument $default, which is a certain sign of a Single Responsibility Principle violation. (undefined)

(BooleanArgumentFlag)